subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -1818,13 +1818,17 @@ static void ticker_cb(uint32_t ticks_at_expire, uint32_t remainder, uint16_t laz
 			uint32_t ticks_interval =
 				HAL_TICKER_US_TO_TICKS((uint64_t)adv->interval *
 						       ADV_INT_UNIT_US);
-			if (adv->ticks_remain_duration > ticks_interval) {
-				adv->ticks_remain_duration -= ticks_interval;
+			uint32_t ticks_elapsed = ticks_interval * (lazy + 1);
```

Comment:
Just so I make sure I understand; this value represents ticks since last adv was sent, and not since advertising was started, right? 

I am simply wondering if it should have been `ticks_elapsed = ticks_interval * adv->event_count`, but it depends on what "elapsed" is referenced to.

Reply:
Ticks elapsed since last radio event, and remaining duration is decremented on every radio event before the radio transmission. This is required so that for a minimum duration, at least once the advertise event is on air.